src/services/notifier/models/subscription.model.ts

```diff
@@ -62,4 +64,10 @@ export default class SubscriptionModel extends Model {
 
   @BelongsTo(() => ProviderModel)
   provider!: ProviderModel
+
+  // @HasOne(() => PlanModel, 'id')
+  // plan!: PlanModel
```

Comment:
Is this comment needed or forgot to remove?

Reply:
🦅👁‍🗨 the latter